### ISSUE TITLE
Doesn't even show prompt

### ISSUE THREAD

**Issue by artivilla99** [CodeBuffet/Slekkel#13]
When I run 'make run' it gives me the following:

```
➜  Slekkel git:(master) ✗ make run
./node_modules/coffee-script/bin/coffee Main.coffee
Please enter your slack details:
```

I'm on a mac os v10.11.1 (El Captain). Let me know what i can look into. Happy to do a screenhero session with you if you'd prefer to just run this on my system!

**Comment by CommandLineDesign** [comment]
I had to highlight the prompt to read the text. I am also using the Solarized Theme for iTerm so I figured the problem was on my end somewhere and it was easy enough to just highlight the questions.

**Comment by artivilla99** [comment]
No, I'm not using iTerm. So here I entered my slack details and I'm not sure what it's prompting me to do next. Any suggestions on what I'm doing wrong here?
<img width="621" alt="screen shot 2015-11-14 at 5 37 18 am" src="https://cloud.githubusercontent.com/assets/2120159/11163122/d96793f0-8a91-11e5-87e6-53bf6a935088.png">

**Comment by ajimix** [comment]
My guess is that you have a problem with your terminal configuration (colors).
